src/utils/fetch-graphql.js

```diff
@@ -329,17 +352,19 @@ ${slackChannelSupportMessage}`
         {
           useVerboseStyle: true,
         }
-      )
+      ),
+      CODES.BadUrl
```

Comment:
In this case I think BadResponse makes more sense. The url might be right but there could be a problem with the WP instance. So this could be BadResponse or HtmlResponse